<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_ETD8A12</class>
 <widget class="QMainWindow" name="MainWindow_ETD8A12">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ETD8A12 Control Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButtonOpen">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBoxDeviceOutputControl">
      <property name="title">
       <string>Device Output Control</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxCh1">
         <property name="text">
          <string>K1B AC</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelCh1">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBoxCh7">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labelCh7">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxCh2">
         <property name="text">
          <string>K1B Lampu</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelCh2">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="checkBoxCh8">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="labelCh8">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxCh3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelCh3">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="checkBoxCh9">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="labelCh9">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBoxCh4">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelCh4">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="checkBoxCh10">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="labelCh10">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBoxCh5">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelCh5">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="checkBoxCh11">
         <property name="text">
          <string>11</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="labelCh11">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBoxCh6">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="labelCh6">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="checkBoxCh12">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="labelCh12">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);background-color: rgb(79, 79, 79);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>OFF</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP Address</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ModBus ID</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBoxInputStatus">
      <property name="title">
       <string>Device Input Status</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labeln1">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labeln2">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labeln3">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labeln4">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labeln5">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labeln6">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN6</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labeln7">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN7</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="labeln8">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN8</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labeln9">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN9</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labeln10">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN10</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labeln11">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN11</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="labeln12">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>21</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>IN12</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButtonReadOutput">
      <property name="text">
       <string>Read Output</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditIPAddress"/>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBoxModbusID">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>25</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>64</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Blok K1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>334</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
